### DISCUSSION TITLE
Can the bullet symbols of unordered lists be customized? Problem with 2nd-level visibility

### DISCUSSION THREAD

**Discussion by pglpm** [quarto-dev/quarto-cli#7195] [Q&A] [answered]
### Description

(Quarto v1.3.450) The Quarto default theme, as well as other themes, use empty circles as bullets for 2nd-level unordered lists. I'm noticing (and some readers have complained) that sometimes they aren't very visible, for instance when they appear in a callout block (owing to the reduced font size there).

So I was wondering whether one can customize the type of list-bullet used. Didn't manage to find anything about this in the manual, nor in a web search.

As a suggestion: maybe another more visible symbol could be used as default for 2nd-level?

Cheers!

**Comment by mcanouil** [comment, marked as answer]
Assuming you are talking about HTML, see <https://www.w3schools.com/CSS/css_list.asp>.
And to properly target the element, use your favourite browser developer/inspect mode.
